exporters/ostream/include/opentelemetry/exporters/ostream/log_exporter.h

```diff
@@ -0,0 +1,129 @@
+        break;
+      case common::AttributeType::TYPE_INT:
+        sout_ << nostd::get<int>(value);
+        break;
+      case common::AttributeType::TYPE_INT64:
+        sout_ << nostd::get<int64_t>(value);
+        break;
+      case common::AttributeType::TYPE_UINT:
+        sout_ << nostd::get<unsigned int>(value);
+        break;
+      case common::AttributeType::TYPE_UINT64:
+        sout_ << nostd::get<uint64_t>(value);
+        break;
+      case common::AttributeType::TYPE_DOUBLE:
+        sout_ << nostd::get<double>(value);
+        break;
+      case common::AttributeType::TYPE_STRING:
+      case common::AttributeType::TYPE_CSTRING:
+        sout_ << nostd::get<nostd::string_view>(value);
+        break;
+
+      /*** Need to support these? ***/
+      case common::AttributeType::TYPE_SPAN_BOOL:
+        sout_ << nostd::get<nostd::span<const bool>>(value);
+        break;
+      case common::AttributeType::TYPE_SPAN_INT:
+        sout_ << nostd::get<nostd::span<const int>>(value);
+        break;
+      case common::AttributeType::TYPE_SPAN_INT64:
+        sout_ << nostd::get<nostd::span<const int64_t>>(value);
+        break;
+      case common::AttributeType::TYPE_SPAN_UINT:
+        sout_ << nostd::get<nostd::span<const unsigned int>>(value);
+        break;
+      case common::AttributeType::TYPE_SPAN_UINT64:
+        sout_ << nostd::get<nostd::span<const uint64_t>>(value);
+        break;
+      case common::AttributeType::TYPE_SPAN_DOUBLE:
+        sout_ << nostd::get<nostd::span<const double>>(value);
+        break;
+      case common::AttributeType::TYPE_SPAN_STRING:
+        sout_ << nostd::get<nostd::span<const nostd::string_view>>(value);
+        break;
+        /******** Up to here ************/
+
+      default:
+        sout_ << "Invalid type";
+        break;
+    }
+  }
+
+  void printKV(nostd::string_view &key, common::AttributeValue &value)
+  {
+    if (firstKV)
+    {
+      firstKV = false;
+    }
+    else
+    {
+      sout_ << ", ";
+    }
+
+    sout_ << "{" << key << ": ";
+    print_value(value);
+    sout_ << "}";
+  }
+
+public:
+  /**
+   * Create an OStreamLogExporter. This constructor takes in a reference to an ostream that the
+   * export() function will send log data into.
+   * The default ostream is set to stdout
+   */
+  explicit OStreamLogExporter(std::ostream &sout = std::cout) noexcept;
+  sdklogs::ExportResult Export(const nostd::span<std::shared_ptr<opentelemetry::logs::LogRecord>>
+                                   &records) noexcept override;
+
+  bool Shutdown(
+      std::chrono::microseconds timeout = std::chrono::microseconds::max()) noexcept override;
+
+private:
+  std::ostream &sout_;
+  bool isShutdown_ = false;
+  bool firstKV     = true;
+
+  // Mapping severity number to severity text from api/include/opentelemetry/logs/log_record.h
+  const std::map<int, std::string> severityMap{
+      {1, "kTrace"},  {2, "kTrace2"},  {3, "kTrace3"},  {4, "kTrace4"},  {5, "kDebug"},
```

Comment:
If this are sequential, contiguous values from 1 to n=24, probably could use const array or vector so doesn't need to do a map lookup.